src/revivalpmmp/pureentities/CustomLogger.php

```diff
@@ -8,11 +8,21 @@
 use pocketmine\utils\TextFormat;
 use pocketmine\Worker;
 
-class CustomLogger extends MainLogger {
+class CustomLogger extends \AttachableThreadedLogger {
```

Comment:
Why not just extend the MainLogger?? It had preexisting functions which don't need remade

Reply:
MainLogger only allows a single existance of itself.  Something about how we were creating the CustomLogger as a child of MainLogger was causing issue #210.  This resolves the problem, still allowing for a custom threaded logger, while we can look for other avenues of approach.